padertorch/contrib/examples/pit/train.py

```diff
@@ -20,36 +21,17 @@
 """
 from sacred import Experiment
 import sacred.commands
-from padercontrib.database.merl_mixtures import MerlMixtures
 import os
 from pathlib import Path
 import padertorch as pt
 import paderbox as pb
+from lazy_dataset.database import JsonDatabase
 
 from sacred.observers.file_storage import FileStorageObserver
 
-from padertorch.contrib.ldrude.data import prepare_iterable
+from padertorch.contrib.examples.pit.data import prepare_iterable
 from padertorch.contrib.ldrude.utils import get_new_folder
-
-
-MAKEFILE_TEMPLATE = """
-SHELL := /bin/bash
-
-train:
-\tpython -m {main_python_path} with config.json
-
-ccsalloc:
-\tccsalloc \\
-\t\t--notifyuser=awe \\
-\t\t--res=rset=1:ncpus=4:gtx1080=1:ompthreads=1 \\
-\t\t--time=100h \\
-\t\t--join \\
-\t\t--stdout=stdout \\
-\t\t--tracefile=%x.%reqid.trace \\
-\t\t-N train_{nickname} \\
-\t\tpython -m {main_python_path} with config.json
-"""
-
+from padertorch.contrib.examples.pit.templates import MAKEFILE_TEMPLATE_TRAIN as MAKEFILE_TEMPLATE
```

Comment:
If possible you should try not to use anything from one of the personal contrib directories here

Reply:
Removing the personal contrib function would make rewriting the example init neccessary. In my opinion, rewriting the example to remove the function would be too time consuming, as all the function does is the automatic creation of an empty folder.